src/Sarif.Converters/FxCopConverter.cs

```diff
@@ -138,6 +140,11 @@ internal Result CreateResult(FxCopLogReader.Context context)
                 sourceFile = string.IsNullOrWhiteSpace(sourceFile) ? targetFile : sourceFile;
             }
 
+            // Don't emit a location if neither physical location nor logical location information
+            // is present. This is the case for CA0001, (unexpected error in analysis tool).
+            // https://docs.microsoft.com/en-us/visualstudio/code-quality/ca0001?view=vs-2019
+            bool emitLocation = false;
```

Comment:
>false [](start = 32, length = 5)

NIT you could make this assignment the return value of !string.IsNullOrWhitespace. Then below, simply test 'emitLocation' to enter the conditional. Eliminates the second assignment to emitLocation.